src/main/webapp/style.css

```diff
@@ -195,6 +195,10 @@ Main Content Container Styling
   color: var(--white);
 }
 
+.unavailable-website {
+  background-color: var(--dark-gray);
```

Comment:
You may also need to disable clicks here, also you might be getting that for free from the browser because there is no url.

Reply:
Disabled clicks!